archunit/src/main/java/com/tngtech/archunit/core/importer/DomainBuilders.java

```diff
@@ -206,6 +206,7 @@ JavaField construct(JavaFieldBuilder builder, ClassesByTypeName importedClasses)
     public abstract static class JavaCodeUnitBuilder<OUTPUT, SELF extends JavaCodeUnitBuilder<OUTPUT, SELF>> extends JavaMemberBuilder<OUTPUT, SELF> {
         private JavaType returnType;
         private List<JavaType> parameters;
+        private List<JavaType> exceptions;
```

Comment:
I would already do the translation here. ASM has `exceptions` but `JavaCodeUnitBuilder` should already have `throwsDeclarations`